app/models/concerns/foreman_remote_execution/exportable.rb

```diff
@@ -12,37 +12,42 @@ module ForemanRemoteExecution
   module Exportable
     extend ActiveSupport::Concern
 
-    def to_export
+    def to_export(include_blank = true)
       self.class.exportable_attributes.keys.inject({}) do |hash, attribute|
-        value = export_attr(attribute, self.class.exportable_attributes[attribute])
+        value = export_attr(attribute, self.class.exportable_attributes[attribute], include_blank)
 
-        # Rails considers false blank, but if a boolean value is explicitly set false, we want to ensure we export it.
-        (value.blank? && value != false) ? hash : hash.update(attribute => value)
+        if include_blank
```

Comment:
I think this whole if is equivalent to

```ruby
hash.update(attribute => value) if include_blank || value.present? || value == false
#or
hash.update(attribute => value) if include_blank || value.present? || value.is_a?(FalseClass)
```

but note that `'    '.present? # => false` as well as `'   '.blank? # => true` which I'm not sure is a problem